consul/base.py

```diff
@@ -1747,12 +1752,16 @@ def create(
             By default the session will be created in the current datacenter
             but an optional *dc* can be provided.
 
+            *token* is an optional `ACL token`_ to apply to this request.
+
             Returns the string *session_id* for the session.
             """
             params = []
             dc = dc or self.agent.dc
             if dc:
                 params.append(('dc', dc))
+            if token:
```

Comment:
```suggestion
            token = token or self.agent.token
            if token:
```
agent may already have token set. Use it!